R/http.R

```diff
@@ -165,8 +173,7 @@ TWIT_paginate_max_id <- function(token, api, params,
       pb$tick()
     }
   }
-
-  results
+  results[1:i]
```

Comment:
You should check that this is needed to avoid a potentially expensive copying of the list.

Reply:
Yes. Turns out this change has had no effect anyways and this works also without it.